### PR TITLE
update: add "Kevin Deng" to core member

### PR DESCRIPTION
**Pull request by Ubugeeei**
## Description of Problem

Kevin Deng is an important figure in the Vue.js community! 
He has made significant contributions to projects such as vuejs/core, Vue Macros, and VueUse.
Recently, he has been active as the lead of vuejs/core-vapor. 
He is definitely someone who should be featured in the documentation! 😄 

## Proposed Solution

Document updation!

### CODE REVIEW

**Comment by Ubugeeei** [comment]
@sxzz 
Please let me know if there are any issues!

File: src/about/team/members-core.json [hunk 1]
```diff
@@ -25,6 +25,38 @@
...
+      {
+        "label": "Vue Macros",
+        "url": "https://github.com/vue-macros/vue-macros"
+      },
+      {
+        "label": "VueUse",
+        "url": "https://github.com/vueuse/vueuse"
+      }
+    ],
+    "location": "Hangzhou, Singapore",
```
Comment (sxzz) [inline src/about/team/members-core.json:49]:
```suggestion
    "location": "Hangzhou 🔁 Singapore",
```

May cause ambiguity

File: src/about/team/members-core.json [hunk 2]
```diff
@@ -25,6 +25,38 @@
     },
     "sponsor": true
   },
+  {
+    "name": "Kevin Deng 三咲智子",
+    "title": "Developer",
```
Comment (sxzz) [inline src/about/team/members-core.json:30]:
```suggestion
    "title": "Open Sourceror",
```

**Review by sxzz — APPROVED** [review]
Thanks bro. But I think I should be added to the end.

**Comment by Ubugeeei** [comment]
Ok.... fixed!

**Pull request merged**